.changeset/rude-parrots-know.md

```diff
@@ -0,0 +1,11 @@
+---
+'@celo/base': major
+'@celo/utils': major
+---
+
+remove deprecated functions and consts exported from ./contacts and ./displayformating. ./currencies and ./phonenumbers. We do not expect this to affect consumers as the use have these functions should have been discontinued and more likely never occured.
```

Comment:
wording isn't the best IMHO, makes us seem unsure

Reply:
```suggestion
remove deprecated functions and consts exported from ./contacts and ./displayformating. ./currencies and ./phonenumbers. If these are used by your app we recommend to inline the functions from the previous release. 

* https://github.com/celo-org/developer-tooling/blob/%40celo/wallet-base%406.0.1/packages/sdk/base/src/contacts.ts 
* https://github.com/celo-org/developer-tooling/blob/%40celo/wallet-base%406.0.1/packages/sdk/base/src/displayFormatting.ts 
* https://github.com/celo-org/developer-tooling/blob/%40celo/wallet-base%406.0.1/packages/sdk/base/src/phoneNumbers.ts

https://github.com/celo-org/developer-tooling/tree/%40celo/wallet-base%406.0.1/packages/sdk/base/src